### DISCUSSION TITLE
能否支持Azure OpenAI？

### DISCUSSION THREAD

**Discussion by CanaanShaw** [binary-husky/gpt_academic#567] [Q&A]
感谢开发者们的工作！我想尝试接入Azure OpenAI APIKey来使用它，不过遇到了一些麻烦。我修改了config.py中的这两行：
API_KEY = "*Azure OpenAI APIKey*"
API_URL_REDIRECT = {"username.openai.azure.com/": "重定向的URL"}

不过在运行时出现了以下错误：
RuntimeError: OpenAI拒绝了请求：{    "error": {        "message": "Incorrect API key provided: 15656127********************41cb. You can find your API key at https://platform.openai.com/account/api-keys.",        "type": "invalid_request_error",        "param": null,        "code": "invalid_api_key"    }}

请问有解决办法吗？感激不尽。烦请见谅如果我的问题解释得太笨拙。
祝生活愉快！

**Comment by binary-husky** [comment]
@CanaanShaw 您好，我们确实没有试过Azure，Azure一直没通过我们的申请，所以没法测试
如果方便配合测试的话请您加一下我qq吧：505030475
